packages/dashboard/src/declarations.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'classnames';
```

Comment:
there is a package for classnames types definition. Can we use that?
https://www.npmjs.com/package/@types/classnames

Reply:
Most definitely. 👍